ui/src/funding/pool.ts

```diff
@@ -244,8 +243,8 @@ export const receiverStore = svelteStore.writable("");
  */
 
 const constraints = {
-  // The constraints for a valid monthly contribution.
-  monthlyContribution: {
+  // The constraints for a valid weekly contribution.
+  weeklyContribution: {
```

Comment:
In the rest of the code we seem to use the term “weekly budget”.

Reply:
Thanks!